NDB.Covid19/NDB.Covid19/Locales/ar.json

```diff
@@ -237,27 +270,27 @@
   "REGISTER_QUESTIONAIRE_ACCESSIBILITY_RADIO_BUTTON_DATEPICKER_TEXT": "اضغط لتحديد تاريخ من التقويم.",
   "REGISTER_QUESTIONAIRE_ACCESSIBILITY_CLOSE_BUTTON_TEXT": "إغلاق",
   "REGISTER_QUESTIONAIRE_CHOOSE_DATE_POP_UP": "اختر التاريخ",
-  "REGISTER_QUESTIONAIRE_RECEIPT_LINK": "https://www.helsenorge.no/en/coronavirus/quarantine-and-isolation/",
+  "REGISTER_QUESTIONAIRE_RECEIPT_LINK": "null",
```

Comment:
why null?

Reply:
Don't know. Copied from sheet

Reply:
I have updated the sheet to contains the link again instead of "null"

Reply:
updated